src/ts/components/subtitleoverlay.ts

```diff
@@ -86,10 +101,89 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
       }
     });
 
+    this.configureCea608Captions(player, uimanager);
     // Init
     subtitleClearHandler();
   }
 
+  configureCea608Captions(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager): void {
+    let ratio = 1;
+    let generateCea608Ratio = () => {
```

Comment:
Rename to `updateCea608FontSize`. This function is not *generating* anything, it is basically calculating the line height and then updating the font size with it.